docs/intro1.md

```diff
@@ -1,15 +1,15 @@
 ---
 id: introduction1
 title: Introduction to Gnosis Protocol
-sidebar_label: Introduction to Gnosis Protocol
+sidebar_label: Introduction
 ---
 Gnosis Protocol is a fully permissionless DEX, which has been in research and development over the course of the last two years. Gnosis Protocol enables ring trades to maximize liquidity. Ring trades are order settlements which share liquidity across all orders, rather than a single token pair, and uniquely suited for trading prediction market tokens and the long tail of all tokenized assets.
 
 Now let's step into the ring, and take a closer look at the basics.
 
 ## Trades
 
-Gnosis Protocol is a trading protocol for ERC-20 tokens. The protocol is technically compatible with any ERC-20 token, but only tokens listed on the protocol can be traded. Token listing is permissionless and can be done by anyone. However, a spam protection fee of 10 <a href="https://blog.gnosis.pm/owl-token-use-cases-6094027ecb37">OWL</a> is required to list new tokens. You can learn more about how to list tokens in this <a href="addtoken1">tutorial</a>.
+Gnosis Protocol is a trading protocol for ERC-20 tokens. The protocol is technically compatible with any ERC-20 token, but only tokens listed on the protocol can be traded. Token listing is permissionless and can be done by anyone. However, a spam protection fee of 10 <a href="https://blog.gnosis.pm/owl-token-use-cases-6094027ecb37">OWL</a> is required to list new tokens. You can learn more about how to list tokens in this <a href="https://docs.gnosis.io/protocol/docs/addtoken1/">tutorial</a>.
```

Comment:
Since you are changing this, would be a nice addition to change `<a href>` to `[](url)` syntax to be in line with the other links